### ISSUE TITLE
Public: change 'Blacklight' to 'UCLA Digital Collections'

### ISSUE THREAD

**Issue by little9** [UCLALibrary/ursus#25]
The internationalization files in Blacklight have been updated to to say 'UCLA Digitial Collections' instead of Blacklight

Child of https://github.com/UCLALibrary/amalgamated-samvera/issues/129
